soc/tenstorrent/tt_blackhole/Kconfig.defconfig.tt_blackhole_smc

```diff
@@ -73,6 +73,9 @@ config 2ND_LVL_INTR_03_OFFSET
 config GPIO
 	default y
 
+config CLOCK_CONTROL
+	default y
```

Comment:
I think we should not enable clock control for SMC until the subsequent integration PR.

```suggestion
```

Reply:
Test it first, and if that's the case, then probably, yea.